es-app/src/CollectionSystemManager.cpp

```diff
@@ -737,6 +737,9 @@ void CollectionSystemManager::populateAutoCollection(CollectionSystemData* sysDa
 						// we may still want to add files we don't want in auto collections in "favorites"
 						include = (*gameIt)->metadata.get("favorite") == "true";
 						break;
+					default:
```

Comment:
Added defaule case to resolve compiler warnings.

/home/devel/temp/EmulationStation_Stable/es-app/src/CollectionSystemManager.cpp:732:12: warning: enum
eration values 'AUTO_ALL_GAMES' and 'CUSTOM_COLLECTION' not handled in switch [-Wswitch]
                                switch(sysDecl.type) {
                                       ^~~~~~~~~~~~
1 warning generated.